ui/app/styles/core.scss

```diff
@@ -126,3 +126,6 @@
 
 // bulma-free-zone
 @import './components/icon';
+
+// dark mode
+@import './core/dark-mode';
```

Comment:
EOF newline?

Reply:
Same nit `ui/app/styles/core/dark-mode.scss`

Reply:
was not aware of this, fixing on next update. thanks